packages/reakit/src/Toolbar/__examples__/ToolbarWithMenu/__tests__/index-test.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import { render, press, click } from "reakit-test-utils";
+import ToolbarWithMenu from "..";
+
+describe("<ToolbarWithMenu />", () => {
+  test("renders toolbar items with closed menu", () => {
+    const { getByText: text } = render(<ToolbarWithMenu />);
+
+    expect(text("Apples")).toBeVisible();
+    expect(text("Oranges")).toBeVisible();
+    expect(text("Other Fruits")).toBeVisible();
+
+    expect(text("Pears")).not.toBeVisible();
+    expect(text("Kiwis")).not.toBeVisible();
+    expect(text("Lemons")).not.toBeVisible();
+  });
+
+  it("can navigate toolbar items through keyboard", () => {
+    const { getByText: text } = render(<ToolbarWithMenu />);
+
+    press.Tab();
+    expect(text("Apples")).toHaveFocus();
+
+    press.ArrowRight();
+    expect(text("Oranges")).toHaveFocus();
+
+    press.ArrowLeft();
+    expect(text("Apples")).toHaveFocus();
+
+    press.ArrowRight();
+    press.ArrowRight();
+    expect(text("Other Fruits")).toHaveFocus();
+
+    press.Enter();
```

Comment:
Nice! 👍
I guess only `press.ArrowDown()` and `press.ArrowUp()` would need `wait()`.